### PR TITLE
Run fonttools against TTF files to save 300KB~

### PR DESCRIPTION
**Pull request by C0rn3j**
Conversation in #471

This needs testing before merge.

### CODE REVIEW

**Review by baxysquare — APPROVED** [review]
I approve of this PR, as it will reduce the font sized in the repository. But I don't currently have an environment to test it in.  We could really use some help from @LibretroAdmin or another top contributor to the main project.

**Review by RobLoach — APPROVED** [review]
Seems to work on Linux with XMB. I don't have any other devices to really test this on, but I assume it's okay?

File: devtools/palettes/pdf/flatux-palette.pdf
Comment (RobLoach) [inline devtools/palettes/pdf/flatux-palette.pdf:1]: You're touching this file in the other PR at https://github.com/libretro/retroarch-assets/pull/472 too... Related?

**Review by RobLoach — CHANGES_REQUESTED** [review]
Looks like a conflict since your last commit?

**Comment by C0rn3j** [comment]
Had to hard reset the fork, but it shouldn't conflict anymore.

**Comment by LibretroAdmin** [comment]
We'll try this. If it causes issues we will have to revert it.

**Pull request merged**